res/css/views/audio_messages/_PlaybackContainer.scss

```diff
@@ -96,13 +85,17 @@ limitations under the License.
                 background-color: transparent;
             }
         }
+    }
+}
 
-        // For timeline-rendered playback, the clock is on the other side of the waveform.
-        & + .mx_Clock {
-            text-align: right;
+.mx_MessageComposer_controls {
```

Comment:
We shouldn't be targeting the composer here - we need to be targeting the PlaybackContainer classes itself.